### PR TITLE
feat: add handout widget

### PR DESCRIPTION
**Pull request by KristinAoki**
Tis PR adds the functionality for the video setting's handout widget.

JIRA Ticket: [TNL-9824](https://2u-internal.atlassian.net/browse/TNL-9824)

### CODE REVIEW

File: src/editors/containers/VideoEditor/components/VideoSettingsModal/components/HandoutWidget/index.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  isUploadError: PropTypes.bool.isRequired,
+  getHandoutDownloadUrl: PropTypes.func.isRequired,
+};
+export const mapStateToProps = (state) => ({
+  handout: selectors.video.handout(state),
+  getHandoutDownloadUrl: selectors.video.getHandoutDownloadUrl(state),
+});
+
+export const mapDispatchToProps = (dispatch) => ({
+  updateField: (stateUpdate) => dispatch(actions.video.updateField(stateUpdate)),
```
Comment (connorhaugh) [inline src/editors/containers/VideoEditor/components/VideoSettingsModal/components/HandoutWidget/index.jsx:122]: nit: stateUpdate implies that this is updating the whole state `payload` is the naming convention we have adopted for modifications to a state (like in a reducer)

**Review by connorhaugh — COMMENTED** [review]
Looks good from a quick review, would love to get another set of eyes on this because it is big. Couple of nits

File: src/editors/data/services/cms/urls.js [hunk 2]
```diff
@@ -50,3 +50,7 @@ export const videoTranscripts = ({ studioEndpointUrl, blockId }) => (
 export const downloadVideoTranscriptURL = ({ studioEndpointUrl, blockId, language }) => (
   `${videoTranscripts({ studioEndpointUrl, blockId })}?language_code=${language}`
 );
+
+export const downloadVideoHandoutUrl = ({ studioEndpointUrl, handout }) => (
+  `${studioEndpointUrl}${handout}`
```
Comment (rayzhou-bit) [inline src/editors/data/services/cms/urls.js:55]: will handout always lead with a `\`?

Comment (rayzhou-bit) [reply src/editors/data/services/cms/urls.js:55]: nvm, I see what's going on now on the api doc

**Review by rayzhou-bit — APPROVED** [review]
I didn't manage to test the code on devstack, but otherwise this looks good to me.

File: src/editors/data/redux/thunkActions/app.js [hunk 3]
```diff
@@ -59,8 +59,8 @@ export const saveBlock = ({ content, returnToUnit }) => (dispatch) => {
 };
 
 export const uploadImage = ({ file, setSelection }) => (dispatch) => {
```
[the remark this answers is not in the document]
Comment (KristinAoki) [reply src/editors/data/redux/thunkActions/app.js:61]: leaving as uploadImage since it is specific to images and the image section of the text editor

**Pull request merged**